NDTensors/ext/examples/NDTensorCUDA.jl

```diff
@@ -1,10 +1,10 @@
-using CUDA
 using NDTensors
-
-using ITensors
-using Test
-
-using Zygote
+using CUDA: cu, CUDA, CuVector
+using ITensors:
+  Index, inner, ITensor, orthogonalize, qr, randomMPO, randomMPS, siteinds, storage, svd
+using Test: @test
+using Zygote: gradient
+using GPUArraysCore: @allowscalar
```

Comment:
```suggestion
```
See comment above.

Reply:
This line has been removed, thanks. I am still working on changing the Metal example.